src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestWithAnnotations.java

```diff
@@ -198,7 +199,6 @@ void doDoubleBraceInitialization() {
             add("AR5E");
         }});} // 2 violations
 
-
```

Comment:
Please remove extra empty line.

Reply:
Done

Reply:
I dropped a word above, please revert removal of empty line, i.e. no diff here.

Reply:
Revert change completely, so that there is no diff here.

Reply:
Please revert this change completely, or remove extra line in separate commit as `minor: remove extra empty line`. This change is not related to this issue.

Reply:
I am sorry. I misunderstood.

Reply:
Resolved, if @romani is good than I am.